README.md

```diff
@@ -112,99 +172,140 @@ The job will appear on the WebUI and you can inspect/restart/kill it.
+    let e_program =
+      KEDSL.Program.shf "echo \"'%s' %s\" | mail -s \"Status update\" %s"
+        cmd sstring
+        email
+    in
+    let e_process =
+      KEDSL.daemonize ~using:`Python_daemon ~host e_program in
+    KEDSL.target ("email result " ^ sstring) ~make:e_process 
   in
-  (* The function `KEDSL.target` creates a node in the workflow graph.
-     This one is very simple, it has a name and a build-process,
-     and since it doesn't have dependencies or fallbacks, it is a
-     “single-node” workflow: *)
-  KEDSL.target
-     "run_command_with_lsf"
-     ~make:lsf_build_process
+
+  (* The function `KEDSL.target` creates a node in the workflow graph. *)
+  KEDSL.target "daemonize command"
+    ~make:build_process
+    ~on_success_activate:[email_target true]
+    ~on_failure_activate:[email_target false]
 
 let () =
-  let workflow =
-     (* Create the  workflow with the first argument of the command line: *)
-     run_command_with_lsf Sys.argv.(1) in
+  (* Grab the command line arguments. *)
+  let cmd   = Sys.argv.(1) in
+  let email = Sys.argv.(2) in
+
+  (* Create the  workflow with the first argument of the command line: *)
+  let workflow = run_command_with_daemonize ~cmd ~email in
+
   (* Then, `Client.submit` is the only function that “does” something, it
      submits the workflow to the engine: *)
   Ketrew.Client.submit workflow
-  (* If Ketrew is in Standalone mode, this means writing the workflow in the
-     database (nothing runs yet, you need to run Ketrew's engine yourself).
-     If Ketrew is in Client-Server mode, this means sending the workflow to the
-     server over HTTPS. The server will start running the workflow right away.  *)
+
 ```
 
-If you actually have access to an LSF cluster and want to try this workflow,
-put it in a file `my_second_workflow.ml`, and simply:
+You can run this [script](src/example_scripts/daemonize_workflow.ml) from the shell
+with
 
-    ocaml my_second_workflow.ml 'du -sh $HOME'
+    ocaml daemonize_workflow.ml 'du -sh $HOME' myaddress@email.com
+    
+Checking in with the gui, we'll have a couple of new tasks:
+
+<div><img width="100%" src="src/doc/images/preview2.png"/></div>
 
 To learn more about the EDSL, you can also explore [examples of more and more
 complicated workflows](src/test/Workflow_Examples.ml) (*work-in-progress*).
 
+Troubleshooting
+---------------
+
+- When reconfiguring `Ketrew` between versions it may be helpful to delete old configurations:
+
+        $ rm -fr $HOME/.ketrew/
+
+- During configuration it is recommended that you pass an authentication token,
+  as opposed to having Ketrew generate one for you:
+
+        $ ketrew init --with-token my-not-so-secret-token
+
+- On a Mac and the EDSL example didn't work? Unfortunately, there's no
+  [setsid](http://man7.org/linux/man-pages/man1/setsid.1.html) utility
+  which is the default way of daemonizing processes in Ketrew.
+  Use the `` `Python_daemon`` technique for forking:
```

Comment:
You're already using the python-hack above, shouldn't this be “if you don't have python on your machine, try nohup-setsid”?